packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -93,21 +93,21 @@ export function createLodestarMetrics(
     }),
     peersRequestedToConnect: register.gauge({
       name: "lodestar_peers_requested_total_to_connect",
-      help: "Priorization results total peers count requested to connect",
+      help: "Prioritization results total peers count requested to connect",
```

Comment:
Here I am not sure what the sentence in `help` means to be honest, the metric name on the other hand makes sense to me. Maybe someone else could clarify this.